test/components/mods/RandomHeldItemsMod.ts

```diff
@@ -0,0 +1,43 @@
+import { INewPokemon } from "./../../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "./../../../src/FullScreenPokemon";
+import { it } from "./../../main";
+import { stubBlankGame } from "./../../utils/fakes";
+
+const pokemonTitle: string[] = "CHARMANDER".split("");
+
+const setUpFSPandGeneratedNumber(generatedNumber: number): FullScreenPokemon {
+    const fsp: FullScreenPokemon = stubBlankGame();
+    fsp.modAttacher.enableMod("Random Held Items");
+    fsp.numberMaker.randomReal1 = (): number => generatedNumber;
+    return fsp;
+}
+
+it("Pokemon spawns with a random item", (): void => {
```

Comment:
> `RandomHeldItemsMod Pokemon spawns with a random item`

Close, but still not quite there.

I like the phrasing you used in the other file:

> `newPokemon gives a new Pokemon a held item`